api/models/task.py

```diff
@@ -657,6 +657,7 @@ class Task(Base):
     lambda_model = db.Column(db.Text)
 
     unique_validators_for_example_tags = db.Column(db.Boolean, default=False)
+    mlcube_tutorial_markdown = db.Column(db.Text)
```

Comment:
We eventually need to define which ORM we're keeping. This should be included as an issue.